test/unit/middleware/partnerapp/handle_gocardless_connect_get_test.js

```diff
@@ -0,0 +1,83 @@
+'use strict'
+
+const getHandler = require('../../../../app/middleware/partnerapp/handle_gocardless_connect_get')
+const baseClient = require('../../../../app/services/clients/base_client/base_client')
+const directDebitClient = require('../../../../app/services/clients/direct_debit_connector_client')
+
+const {expect} = require('chai')
+const sinon = require('sinon')
+
+let req, res, stubbedBaseClientPost, stubbedGetGatewayAccount
+
+describe('When GoCardless Connect middleware receives a GET request', function () {
+  beforeEach(function () {
+    req = {
+      account: {
+        type: 'test'
+      },
+      query: {
+        state: 'some-test-state',
+        code: 'a-test-code'
+      }
+    }
+
+    res = {
+      end: () => {
+        expect(true).to.equal(true)
+      }
+    }
+
+    stubbedBaseClientPost = baseClient.post = sinon.stub()
```

Comment:
I like the initialisation of the two variables in one line :) 👍 
I guess we are creating `stubbedBaseClientPost` just for better readability

Reply:
Yep, for readability, can be removed, depends on taste :)